lib/wraft_doc_web/router.ex

```diff
@@ -104,6 +104,14 @@ defmodule WraftDocWeb.Router do
       # Set Password for first time
       post("/users/set_password", UserController, :set_password)
     end
+
+    # templates
+    scope "/v1", Api.V1, as: :v1 do
+      scope "/templates" do
+        get("/", TemplateAssetController, :list_public_templates)
```

Comment:
template_assets/public might be better as it wont clash with the existing route for template_assets index API coming defaultly. Just a suggestion.